.gitignore

```diff
@@ -12,3 +12,5 @@ coverage-final.json
 coverage-final.lcov
 npm-debug.log
 yarn.lock
+/src/styles/dgrid.css.d.ts
```

Comment:
Talked about this with Matt, and while he thinks this is okay for now, he'd like to move this sort of CSS wrapper into the build, or move this file outside of the src directory and copy it during the build.